app/view/button/ShowMeasureToolsPanelModel.js

```diff
@@ -0,0 +1,10 @@
+Ext.define('MoMo.client.view.button.ShowMeasureToolsPanelModel', {
+    extend: 'Ext.app.ViewModel',
+
+    alias: 'viewmodel.button.showmeasuretoolspanel',
+
+    data: {
+        tooltip: 'Messwerkzeuge',
```

Comment:
I've just flown across the code - it's really not consistently at the moment :tired_face: Some tools are in german, some in english, admin and login sites are completely in english..

I've just created an issue with this problem --> https://github.com/terrestris/momo3-frontend/issues/30